manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "A self-hosted financial manager.",
         "fr": "Un gestionnaire de finances personnelles."
     },
-    "version": "5.2.0~ynh2",
```

Comment:
You should add an upgrade test from 5.2.0~ynh2 to check_process

Reply:
I have added the hash for it in check_process.